app/controllers/api/subcollections/alert_actions.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module Subcollections
+    module AlertActions
+      def alert_actions_query_resource(object)
+        MiqAlertStatusAction.where(:miq_alert_status_id => object.id)
```

Comment:
Do alert statuses not have alert status actions? i.e. something like `object.miq_alert_status_actions`?